lib/openstudio-standards/standards/ashrae_90_1_prm/userdata_csv/ashrae_90_1_prm.UserData.rb

```diff
@@ -274,7 +279,7 @@ class UserDataWaterUseConnection < UserDataCSV
   # @param save_dir [String] directory to save user data files
   def initialize(model, save_dir)
     super
-    @component_name = 'WaterUseConnections'
+    @component_name = 'WaterUseConnectionss'
```

Comment:
Missing an 's' here. Without it, you're asking for another function, that requires a "handle" parameter.